### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jjkoehorst**
Added make info at the end of the documentation related to useful make commands when developing for cwltool. Please check if the description is correct as I am not too familiar with make yet.

### CODE REVIEW

**Review by mr-c — APPROVED** [review]
Thank you very much @jjkoehorst ! I was inspired by your work and I added many additions.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,32 +1,68 @@
...
+that cannot be fixed automatically.
+
+When tests are passing, you can create git commits in a new branch and push to
+GitHub to make a pull request on `cwltool` repo
+
+Useful `make` commands:
+The `make` commands below can help you to install the dependencies, format and test your code
+
+- `make help` to show the list of commands that can be used with the make function
+- `make install-dep` will install the dependencies needed by cwltools
```
Comment (kinow) [inline CONTRIBUTING.md:56]: s/cwltools/cwltool ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,32 +1,68 @@
...
+- `make format` will clean up your code according to the accepted python standard
+- `make test` will perform the tests of cwltools (can take a while to run all tests)
```
Comment (kinow) [inline CONTRIBUTING.md:58]: :point_up:

File: CONTRIBUTING.md (hunk 1 above)
Comment (mr-c) [inline CONTRIBUTING.md:56]:
```suggestion
- `make install-dep` will install the dependencies needed by `cwltool`
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (mr-c) [reply CONTRIBUTING.md:58]:
```suggestion
- `make test` will perform the tests of `cwltool` (can take a while to run all tests)
```

**Review by kinow — APPROVED** [review]
:clap:

**Pull request merged**